model/countries/global/global-descriptions.yaml

```diff
@@ -1,5 +1,5 @@
 short-descriptions:
-  full-address: Artificial concept, not to be used in HTML
+  address: Address of a physical location but without a name - Artificial concept, not to be used in HTML
```

Comment:
Can probably drop the "but without a name" part.

Reply:
Once more change I remembered. The model for IN contains locality2 twice. It exist under `street-address-alternative-1` and `address`.

Reply:
Yes... the modeling for India will require more thoughts.